### DISCUSSION TITLE
dump environment in hello world type test jobs

### DISCUSSION THREAD

**Discussion by marianne013** [DIRACGrid/DIRAC#6517] [Q&A] [answered]
Hi,

I was hoping to add minimal debug output (aka dumping the environment) to the hello world job(s) we use in the hackathon, but I figured this might have side effects elsewhere, so I was wondering what the best way forward would be.
I note that in 
DIRACRepo/src/DIRAC/tests/Workflow/Integration/
there are two files:
exe-script.py  and helloWorld.py
As far as I can tell the test jobs all use exe-script.py.
I wanted to add a line:
```
# Main
if __name__ == "__main__":
    subprocess.call(shlex.split("env")) <----
    subprocess.call(shlex.split("echo Hello World")) 
    sys.exit(0)
```
Would it make sense to add this to exe-script.py or will that break stuff ?
Or is anyone using helloWorld.py ? If not I could modify this and have the cloud tests use this instead.
Or should I make a completely new file for the cloud tests ?
I'm just not sure what the rationale behind the "Hello World" tests is.

Daniela

**Comment by fstagni** [comment]
helloWorld.py is used in Jenkins tests. 

I don't see anything wrong in you adding call to `env`.

**Comment by marianne013** [comment, marked as answer]
Doing the end of year open issue clean up, created pull request:
https://github.com/DIRACGrid/DIRAC/pull/6636
